docs/topics/multiplatform/multiplatform-hierarchy.md

```diff
@@ -64,8 +64,10 @@ are no watchOS targets in the project.
 If you add a watchOS target, like `watchosArm64`, the `watchos` source set is created, and the code
 from the `apple`, `native`, and `common` source sets is compiled to `watchosArm64` as well.
 
-The Kotlin Gradle plugin creates type-safe accessors for all of the source sets from the default hierarchy template,
-so you can reference them without `by getting` or `by creating` constructions compared to the [manual configuration](#manual-configuration):
+The Kotlin Gradle plugin provides both type-safe and static accessors for all of the source sets from the default hierarchy
+template, so you can reference them without `by getting` or `by creating` constructions compared to the [manual configuration](#manual-configuration).
```

Comment:
```suggestion
template, so you can reference them without `by getting` or `by creating` constructs compared to the [manual configuration](#manual-configuration).
```

I don't believe "construction" has the meaning needed here, but [language constructs](https://en.wikipedia.org/wiki/Language_construct) are definitely a thing.

Reply:
As I see on our Coding conventions page, it's used as a synonym to "constructs", plus a few other mentions in the docs. But we can of course change it here, no problem